homeassistant/components/sensor/modem_callerid.py

```diff
@@ -18,7 +18,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 DEFAULT_NAME = 'Modem CallerID'
-ICON = 'mdi:phone-clasic'
+ICON = 'mdi:phone-classic'
```

Comment:
This probably doesn't break anything (only fix something), but changes the semantics of the codebase.